index.js

```diff
@@ -43,7 +43,16 @@ class MarkdownButtonElement extends HTMLElement {
 class MarkdownHeaderButtonElement extends MarkdownButtonElement {
   constructor() {
     super()
-    styles.set(this, {prefix: '### '})
+
+    if (!this.hasAttribute('level')) {
+      this.setAttribute('level', '3')
```

Comment:
I don't think there's any need to set the attribute if it's not set. Just defaulting to 3 would be good.

Reply:
Yeah I went back and forward on this, will put it back to no attribute by default